cli/tauri.js/test/jest/__tests__/dev.spec.js

```diff
@@ -48,9 +57,10 @@ describe('Tauri Dev', () => {
 
   const devServer = startDevServer()
 
-  for (const devPath of [devServer.url, distDir]) {
-    const runningDevServer = devPath.startsWith('http')
-    it(`works with the ${runningDevServer ? 'dev server' : 'dev path'} mode`, () => {
+  test.each([[devServer.url], [distDir]])(
```

Comment:
You can use `it.each` instead, that would stay in line with the rest of the tests.